Kernel/TTY/VirtualConsole.cpp

```diff
@@ -302,6 +302,16 @@ void VirtualConsole::flush_dirty_lines()
             continue;
         for (size_t column = 0; column < columns(); ++column) {
             auto& cell = cell_at(column, visual_row);
+            if (cell.attribute.is_untouched()) {
```

Comment:
The `touched` flag is set when something is written into the cell from its pristine state (it's essentially "dirtied" but lives across rerenders, _forever_).
This code seems fine, an untouched cell has never had anything written to it, so it should remain black.

Reply:
I just wonder why I see a gray background screen for about 2 seconds on boot in text mode console. Maybe someone or something initializes the entire cell region to have a gray color, but that's still weird...

Reply:
Do we maybe forget to initialize the cells? If that happens, `terminal_to_standard_color` will find garbage instead of proper color values, and will fall back to light gray:

https://github.com/SerenityOS/serenity/blob/065525aba084b3ba7a7932615be72cd43b5c10da/Kernel/TTY/VirtualConsole.cpp#L233-L241